terraform-plans/configs/hardware-observer-operator_main.tfvars

```diff
@@ -21,7 +21,9 @@ templates = {
     source      = "./templates/github/charm_check.yaml.tftpl"
     destination = ".github/workflows/check.yaml"
     vars = {
-      runs_on            = "[[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]",
+      # Cannot test on s390x because setup-python action does not support s390x
+      tests_on           = "[[ubuntu-24.04], [Ubuntu_ARM64_4C_16G_01]]",
+      builds_on          = "[[ubuntu-24.04], [Ubuntu_ARM64_4C_16G_01], [self-hosted, linux, s390x]]",
```

Comment:
I have a concern about this. Prometheus-hardware-observer is a python package and I'm not sure that  tools like lshw, dmidecode, nvme will work as we expect. Moreover, auxiliary snaps for hwo like dcgm, smartctl will need to start building and releasing snaps in this arch whic I don't see here.

Maybe is too soon to start creating this charm for this arch without we investigate.

Reply:
None of the charms listed here are tested. AFAIK, they only need to be installable (error or not, as long as it is not blocked by `juju`). @Pjack can you confirm the goal?

Reply:
Yes, our primary goal is to ensure it’s installable first. The testing environment won’t be available until PS7 ready. But it's also correct that we should include dcgm and the smartctl snap in the scope if possible.